content/dstu2/contract.md

```diff
@@ -0,0 +1,66 @@
+
+## Terminology Bindings
+
+<%= terminology_table(:contract, :dstu2) %>
+
+## Search
+
+Search for Contracts that meet supplied query parameters:
+
+    GET /Contract?:parameters
+
+### Parameters
+
+ Name                 | Required?                                                    | Type          | Description
+----------------------|--------------------------------------------------------------|---------------|------------------------------------------------------------------------------------------------------------------------------
+ `_id`                | No, if populated all other parameters are ignored            | [`token`]     | The logical resource id associated with the resource.
+ `actor`              | Yes, or `_id`                                                | [`reference`] | The actor (RelatedPerson) of the Contract. Example: `actor=RelatedPerson/1234` or `actor:RelatedPerson=1234`
+ `actor.identifier`   | Yes, or `_id`                                                | [`token`]     | The identifier for the actor of the Contract. Example `actor.identifier=TODO`
+ `patient.identifier` | Yes, or `_id`                                                | [`token`]     | The identifier for the patient subject of the Contract. Example `actor.identifier=TODO`
+ `subject`            | Yes, or `_id`                                                | [`reference`] | The subject (Patient) of the Contract. Example `subject=Patient/1234` or `subject:Patient=1234`
+ [`_include`]         | No                                                           | [`string`]    | Other resource entries to be returned as part of the bundle. Example `_include=Contract:actor` or `_include=Contract:patient`
+
+Notes: 
+
+- Either the `_id` or one of `actor`, `actor.identifier`, `patient.identifier`, or `subject` parameters must be provided.
+- The `actor`, `actor.identifier`, `patient.identifier`, or `subject` parameters may be provided exactly once and may have only a single value.
+- The `_include` parameter may be provided once for each resource to be included. Example: `_include=Contract:actor&_include=Contract:patient`
+- Additional OAuth2 scopes are required when the `_include` parameter is provided. To include with `Contract:actor` or `Contract:patient` values, the OAuth2 token must include scopes for one or both RelatedPerson.read and Patient.read, respectively, depending on the combination of resources to be included.
+
+### Response
+
+Search for all Contracts subject to a specific patient
+<%= headers 200, GET: '[...]/Contract?subject:Patient=14067775' %>
```

Comment:
Leaving as-is for now. This is reusing existing code and it's not quite as simple as just flipping the order. We can revisit in the future if necessary